### PR TITLE
chore: add readme quick start

### PR DESCRIPTION
**Pull request by dskamiotis**
## What does this change?

We do not have a general quick start guide to help all users of prosemirror-elelements to quickly and easily create an element.

## How to test

Check the doc in its markdown form for both structure and content for accuracy :)

Checked the layout in https://dillinger.io/

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+```
+
+## Where to keep our element
+By convention, we’ve defined and kept our elements in `src/elements`. Let’s add our pullquote there, in `src/elements/example-pullquote/ExamplePullquote.tsx`, and export the element we’ve just created:
+```js
+export const pullquoteElement = createReactElementSpec(
+  // …
+```
+
+# Creating the Element Form
```
Comment (jonathonherbert) [inline docs/quick-start.md:170]: Is this section still necessary? We've introduced these concepts further up.

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+# Adding a new element using ProseMirror Elements
+## Description
+The purpose of this document is to give a quick and easy guide to creating an element for a rich text editor using prosemirror-elements.
+
+In this example we will demonstrating this using React with Typescript and utilising the array of functions, types and design patterns that prosemirror-elements gives us for free.
```
Comment (jonathonherbert) [inline docs/quick-start.md:5]:
```suggestion
We'll do this using React and Typescript, with the functions, types and React components that prosemirror-elements gives us for free.
```

File: docs/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+What is a Field?
+Each Element is made up of Fields, which represent a data type – for example, text, rich text, or custom data types. 
+
+For example, a Pullquote Element might contain the following fields:
+
+A richText field for styled pullquote content.
+A text field for the attribution.
+A custom field, role, containing type `string` to indicate the pullquote’s presentation.
+A custom field, isMandatory, containing type `boolean` to indicate whether the pullquote should always be displayed.
+# Example: creating a pullquote Element in React
```
Comment (jonathonherbert) [inline docs/quick-start.md:22]:
```suggestion
# Example: creating a pullquote Element in React
```

By convention, we should only have one h1 (`#`) on the page, and everything else should be h2 (`##`) or smaller.

**Review by jonathonherbert — APPROVED** [review]
Great start! A few comments. We can expand these docs further to include mounting the component in the demo once that's in a clearer state.

File: docs/quick-start.md (hunk 1 above)
Comment (dskamiotis) [reply docs/quick-start.md:170]: Of course we have done this in 1 file

**Review by jonathonherbert — APPROVED** [review]

**Pull request merged**